packages/nimble-components/src/select/template.ts

```diff
@@ -119,7 +119,7 @@ SelectOptions
                         class="scrollable-region">
                         <slot
                             ${slotted({
-                                filter: (n: Node) => n instanceof HTMLElement && Listbox.slottedOptionFilter(n),
+                                filter: (n: Node) => n instanceof HTMLElement && isListboxOption(n),
```

Comment:
Need both?

```suggestion
                                filter: (n: Node) => isListboxOption(n),
```

[isListboxOption](https://github.com/microsoft/fast/blob/archives/fast-element-1/packages/web-components/fast-foundation/src/listbox-option/listbox-option.ts#L22) [isHTMLElement](https://github.com/microsoft/fast/blob/archives/fast-element-1/packages/utilities/fast-web-utilities/src/dom.ts#L6)

Reply:
Sorry...this doesn't work. The `instanceof` check allows for it to be passed to the `isListboxOption` method which expects an `Element` type.